src/include/ZividPython/Wrappers.h

```diff
@@ -13,6 +16,50 @@
 
 namespace ZividPython
 {
+    namespace
+    {
+        std::string toSnakeCase(const std::string capitalized)
```

Comment:
If the variable name is supposed to describe the casing style, perhaps it should be changed from `capitalized` to `upperCamelCase` (which unfortunately itself has to be written in lower camel case ;) )
https://en.wikipedia.org/wiki/Camel_case

Reply:
Should be fixed now